lib/plugin.js

```diff
@@ -44,38 +44,67 @@ exports.register = function (server, opts, next) {
     }
 
     const getById = function (schema) {
-        onRouteRegister(schema);
+        onRouteRegister(schema)
+        server.route({
```

Comment:
Can we move these to a util function? so that we can have small functions on this module and better see what the flow is.